src/test/cart.test.js

```diff
@@ -52,3 +52,28 @@ describe('Product already in cart',() => {
         });
   });
 });
+
+// describe('Viewing products on cart',() => {
```

Comment:
Remove commented codes if they are not needed, if those tests are needed uncomment them and refine